dom.bs

```diff
@@ -2389,12 +2389,10 @@ of a <var>node</var> into a <var>parent</var> before a <var>child</var>, run the
 </ol>
 
 <p><a lt="Other applicable specifications">Specifications</a> may define
-<dfn export id=concept-node-insert-ext>insertion steps</dfn> for all or some <a for=/>nodes</a>. The
-algorithm is passed <var ignore>insertedNode</var>, as indicated in the <a for=/>insert</a>
-algorithm below.
-<!-- See https://github.com/whatwg/dom/issues/34#issuecomment-125571750 for why we might need to
-     adjust this further based on the requirements of the script element. There might be other ways
-     to define that though as Olli suggests, so leaving that out for now. -->
+<dfn export id=concept-node-insert-ext>insertion steps</dfn> or
+<dfn export id=concept-node-children-added>children added steps</dfn> for all or some
```

Comment:
So I'm pretty sure implementations have a "children changed steps" callback. Removing children does not affect this? What if someone modifies a `Text` node directly?

I'm pretty sure we should merge "children added steps" and "child text content change steps".

The main place that might require some detailed tests around this is the `style` element. As far as I can tell it's mostly not observable for `textarea`. I could write some tests around this if that would help.

Reply:
I've created #800 to add "children changed steps". Once that's accepted I'll rebase this accordingly and see what's left.